inference-engine/tests/functional/plugin/cpu/test_utils/fusing_test_utils.hpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_test_utils.hpp"
+#include "single_layer_tests/activation.hpp"
+
+namespace FusingTestUtils {
+
+struct postNode {
+    std::shared_ptr<ngraph::Node> nodePtr;
+    std::map<std::string, std::string> addInfo;
+};
+
+typedef std::tuple<
+        std::shared_ptr<ngraph::Function>,
+        std::vector<postNode>,
+        std::vector<std::string>> fusingSpecificParams;
+
+const std::vector<size_t> fakeShape = {1};
+const auto fakeConstNode = ngraph::builder::makeConstant(ngraph::element::f32, fakeShape, {}, true);
+
+/* FUSING PATTERNS */
+const auto fusingRelu = fusingSpecificParams{nullptr,
+        {{ngraph::builder::makeActivation(fakeConstNode, ngraph::element::f32, ngraph::helpers::Relu)}}, {"Relu"}};
```

Comment:
Are you sure that static construction of "hard" ngraph object in static section is a good idea?
It will be executed each time even you no need it...
I prefer to move it into runtime stage